src/UKHO.ERPFacade.Common/Models/LicenceUpdatedEventPayload.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+
+namespace UKHO.ERPFacade.Common.Models
+{
+    [ExcludeFromCodeCoverage]
+    public class LicenceUpdatedEventPayload
```

Comment:
It seems that mentioned file is not a part of this PBI. please remove it.

Reply:
Removed file.